components/builder-api/src/server/mod.rs

```diff
@@ -118,6 +119,7 @@ impl AppState {
 fn enable_features(config: &Config) {
     let features: HashMap<_, _> = HashMap::from_iter(vec![("LIST", feat::List),
                                                           ("JOBSRV", feat::Jobsrv),
+                                                          ("OPKGSETTINGS", feat::OpkgSettings),
```

Comment:
Minor, but might be good to rename this to something like "LEGACYPKG".. once we are stable we can just remove it

Reply:
:+1: